assets.go

```diff
@@ -84,10 +82,6 @@ func (l *Loader) Json(name string) string {
 	return l.jsons[name]
 }
 
-func (l *Loader) Level(name string) *Level {
-	return l.levels[name]
-}
-
```

Comment:
Removed but may later be added. It refers to `Level`, something available only in `core` (and this is `engo`, which cannot import `core`).